src/games/calculator.js

```diff
@@ -0,0 +1,20 @@
+import { getAnswer, askQuestion } from '../cli.js';
+import { operators, generateRandomNumber } from '../utils/math.js';
+
+const calculator = () => {
+  const operator = Object.keys(operators)[generateRandomNumber(Object.keys(operators).length - 1)];
+
+  const leftOperand = generateRandomNumber(100);
+  const rightOperand = generateRandomNumber(operator === '*' ? 10 : 100);
+
+  askQuestion(`${leftOperand} ${operator} ${rightOperand}`);
+
+  const correctAnswer = operators[operator](leftOperand, rightOperand);
+  const answer = getAnswer();
+
+  const guess = Number(answer) ? Number(answer) : answer;
```

Comment:
Too much stuff happens on this line I guess. You're checking if `Number(answer)` is NaN and if so, you return a converted number. It's better to make this check and behaviour explicit. Moreover, expecting NaN input doesn't make sense in this game.

```suggestion
if (isNaN(guess)) {
  // throw an exception, or show a message to user
}

const guess = Number(answer)
```